<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(133, 92, 119);</string>
  </property>
  <widget class="QWidget" name="Pomodoro" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(203, 34, 79);
color:rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(195, 141, 179, 89);
border-radius:30px;</string>
    </property>
    <property name="text">
     <string>25:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="addTaskWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(195, 141, 179, 89);
border-radius:30px;
</string>
    </property>
    <widget class="QLabel" name="addYourTasksText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Add your task(s) for this Pomodoro session</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="taskInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(217, 217, 217, 66);
border-radius:10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addTask">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Add Task</string>
     </property>
    </widget>
    <widget class="QLabel" name="yourTasksText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Your tasks:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="tasksCombo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="pomodoroSession">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>20</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Pomodoro Session</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfSession">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>40</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startStopButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doneButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>I'am done with this session</string>
    </property>
   </widget>
   <widget class="QWidget" name="notFinishedButton" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>581</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(195, 141, 179, 89);
border-radius:30px;
</string>
    </property>
    <widget class="QLabel" name="tasksBelow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>14</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Tasks:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="labelAsNotFinishedButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Chalkboard</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Label as not finished</string>
     </property>
    </widget>
    <widget class="QComboBox" name="tasksCombo_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="goToMainMenuButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkboard</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
border-radius:10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
